.changeset/shiny-badgers-film.md

```diff
@@ -0,0 +1,8 @@
+---
+"amplify-ui-angular-mono": patch
```

Comment:
Can you remove `amplify-ui-angular-mono`? This is a private dev folder, but changeset adds it by default 😓